tests/test_location_pi.py

```diff
@@ -263,7 +298,36 @@ def test09_process_pi_count_moves_valid_request_with_quant(self):
                          self.package_one.id,
                          "Picking is not related to the specified package")
 
-    def test10_process_pi_count_moves_bad_request_with_reserved_package(self):
+    def test11_process_pi_count_moves_valid_request_with_two_quants(self):
+        """
+        Successfully creates the expected picking when two quants are
+        specified (two quants for different products, same package);
+        expecting a picking with 2 move lines.
+        """
+        quant01 = self.create_quant(self.apple.id, self.test_location_01.id,
+                                    13, package_id=self.package_one.id)
+        quant02 = self.create_quant(self.banana.id, self.test_location_01.id,
+                                    11, package_id=self.package_one.id)
+        count_moves = [{"quant_ids": [quant01.id, quant02.id],
+                        "location_id": self.test_location_01,
+                        "location_dest_id": self.test_location_02.id}]
+
+        pickings = self.test_location_01._process_pi_count_moves(count_moves)
+
+        self.assertEqual(len(pickings), 1,
+                         "Did not create exactly one picking")
+
+        self._check_picking_loc01_to_loc02(pickings[0], 0,
+                                           check_single_move_line=False)
+        self.assertEqual(len(pickings[0].move_line_ids), 2,
+                         "Did not create a picking with 2 move lines")
+
+        for idx in range(2):
```

Comment:
We can use for `move_line in pickings[0].move_line_ids:` since we don't need idx for anything.